pvt/helpers/src/models/pools/weighted/WeightedPool.ts

```diff
@@ -17,6 +17,7 @@ import {
   InitWeightedPool,
   JoinGivenInWeightedPool,
   JoinGivenOutWeightedPool,
+  MultiJoinGivenOutWeightedPool,
```

Comment:
```suggestion
  JoinAllGivenOutWeightedPool,
```

Reply:
'multi' makes it sound like you can choose which tokens to use

Reply:
Agree that's better - I had it Multi for consistency with MultiExit. Should MultiExit also be ExitAll then?

Reply:
No, let's not change any old names unless we have to. Given that join and exit kinds are not _that_ related, I wouldn't worry about exit kinds are all.